DiceKeys/Views/Screens/DiceKeyPresent.swift

```diff
@@ -26,49 +26,56 @@ struct DiceKeyPresentNavigationFooter: View {
 
     var body: some View {
         let view = VStack {
         ZStack {
             HStack(alignment: .top) {
-                
+                Spacer()
                 VStack {
                     Image("USB Key")
                         .renderingMode(.template)
                         .resizable()
                         .aspectRatio(contentMode: .fit)
-                        .frame(height: min(geometry.size.width, geometry.size.height)/10, alignment: .center)
+//                        .frame(height: navBarImageHeight, alignment: .center)
                         .onTapGesture { navigateTo(.SeedHardwareKey) }
 //                    Button(action: {  }) {
 //                        VStack {
                             Text("Seed a SoloKey").multilineTextAlignment(.center).font(.footnote)
 //                        }
 //                    }
-                }.frame(width: geometry.size.width * BottomButtonFractionalWidth, alignment: .center)
+
+                }.frame(width: geometry.size.width * BottomButtonFractionalWidth,
+                        height: navBarContentHeight, alignment: .center)
                 .onTapGesture {
                     navigateTo(.SeedHardwareKey)
                 }
-                .if( pageContent == .SeedHardwareKey ) { $0.colorInvert() }
-                VStack {
+                .if( pageContent == .SeedHardwareKey ) { $0.colorInvert()
+                }
+                DerivationRecipeMenu({ newPageContent in
+                    navigateTo(newPageContent)
+                }) {
                     VStack {
                         Image(systemName: "arrow.down")
                             .resizable()
                             .aspectRatio(contentMode: .fit)
+                        Spacer(minLength: 2)
                         Image(systemName: "ellipsis.rectangle.fill")
                             .resizable()
                             .aspectRatio(contentMode: .fit)
-                    }.frame(height: min(geometry.size.width, geometry.size.height)/10, alignment: .center)
-                    DerivationRecipeMenu({ newPageContent in
-                        navigateTo(newPageContent)
-                    }) {
-                        HStack {
-                            Spacer()
-                            Text("Derive a Secret").multilineTextAlignment(.center).font(.footnote)
-                            Spacer()
-                        }
-                    }
+                        Spacer(minLength: 2)
+                        Text("Derive a Secret").multilineTextAlignment(.center).font(.footnote)
+                    }.frame(width: geometry.size.width * BottomButtonFractionalWidth,
+                            height: navBarContentHeight)
                 }
-                .frame(width: geometry.size.width * BottomButtonFractionalWidth, alignment: .center)
+                .frame(width: geometry.size.width * BottomButtonFractionalWidth,
```

Comment:
We probably don't need to set both frames. Remove the frame in line 71 if we keep this one.

Reply:
Fixed.